content/webapp/pages/exhibition-guide.tsx

```diff
@@ -286,8 +428,29 @@ const ExhibitionGuidesPage: FC<Props> = props => {
           </Space>
         </>
       )}
+      {otherExhibitionGuides.results.length > 0 && (
+        <PromoContainer>
+          <Space
+            v={{ size: 'xl', properties: ['padding-top', 'padding-bottom'] }}
+          >
+            <Layout8 shift={false}>
+              <Space v={{ size: 'l', properties: ['margin-bottom'] }}>
+                <h2 className="h2">Other exhibition guides available</h2>
+              </Space>
+            </Layout8>
+            <CardGrid
+              itemsHaveTransparentBackground={true}
+              items={otherExhibitionGuides.results.map(result => ({
+                ...result,
+                type: 'exhibition-guides-links',
```

Comment:
This makes me feel a bit bleugh, but seemed the simplest way to not have to duplicate all the layout logic in the CardGrid component